pyscitt/pyscitt/prefix_tree.py

```diff
@@ -104,7 +104,7 @@ def from_cose_obj(cls, cose_obj: list) -> "ReadReceipt":
 
     @classmethod
     def decode(cls, data: bytes):
-        return cls.from_cose_obj(cbor2.loads(data))
+        return cls.from_cose_obj(cbor2.loads(data))  # type: ignore[arg-type]
```

Comment:
Just for my understanding, why did we add all these type check ignores in other files?

Reply:
because the checks started to fail out of the blue, I believe this is to do with the unpinned dependencies

Reply:
What i am trying to say is that nothing changed in the logic so I just ignored these as it seemed like noise